README.md

```diff
@@ -7,11 +7,17 @@ To correctly connect to the provided database and run this project, it is requir
 3- npm install dependencies.
 4- run npm setup.
 5 -run npm seed.
-6- You are know succesfully connected to the database
+6- You are now succesfully connected to the database.
+7- To run the tests just type run rpm test int.
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 Branch: 2-get-topics
```

Comment:
I don't think describing your branches is necessary for your readme, there is a later ticket which will ask you to provide a more descriptive readme for potential users, but for now it's fine just with information about how to run the api